### PR TITLE
Release 2022.12.2

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Release note branch.
- **Is this PR related to a Monolith release?** Yes. 2022.12.2

### Resolves
* [OKTA-559421](https://oktainc.atlassian.net/browse/OKTA-559421)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 1]
```diff
@@ -6,11 +6,25 @@ title: Okta Identity Engine API Products release notes 2022
...
 
+### Weekly release 2022.12.2
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                      |December 21, 2022            |
+
+#### Bugs fixed in 2022.12.2
+
+* UPDATE requests to the `/brands/{brandId}/pages/sign-in/preview` and `/brands/{brandId}/pages/sign-in` endpoints sometimes returned a `null` response that resulted in an HTTP 400 Bad Request error. (OKTA-551632)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:17]:
```suggestion
* PUT requests to the `/brands/{brandId}/pages/sign-in/preview` and `/brands/{brandId}/pages/sign-in` endpoints under certain conditions returned a `null` response body with an HTTP 400 Bad Request error. (OKTA-551632)
```
Sorry, I know this has already been reviewed, but you shouldn't capitalize UPDATE unless it's a REST request operations? Maybe use "Update requests to the ..." or use "PUT requests to the ..."?
Also, the null response doesn't "resulted" an 400 bad request error? Please confirm with dev.

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 2]
```diff
@@ -6,11 +6,25 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+* If an admin added redirect URIs and reached the character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:19]:
```suggestion
* If an admin added redirect URIs that reached the maximum character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
```
Not sure if this edit is ok?

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 3]
```diff
@@ -6,11 +6,25 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+* When an admin viewed a user's profile, an error sometimes occurred. (OKTA-558220)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:21]: Can we be more specific from dev perspective? This seems like an Admin Console bug? What's the API request?

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 4]
```diff
@@ -4,11 +4,23 @@ title: Okta API Products release notes 2022
...
 
+### Weekly release 2022.12.2
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                         |December 21, 2022            |
+
+#### Bugs fixed in 2022.12.2
+
+* UPDATE requests to the `/brands/{brandId}/pages/sign-in/preview` and `/brands/{brandId}/pages/sign-in` endpoints sometimes returned a `null` response that resulted in an HTTP 400 Bad Request error. (OKTA-551632)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:15]: same comment from before

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 5]
```diff
@@ -4,11 +4,23 @@ title: Okta API Products release notes 2022
...
+
+* If an admin added redirect URIs and reached the character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:17]:
```suggestion
* If an admin added redirect URIs that reached the maximum character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
```
same comment as above

**Review by vanngo-okta — CHANGES_REQUESTED** [review]
Some questions for you

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 6]
```diff
@@ -6,11 +6,25 @@ title: Okta Identity Engine API Products release notes 2022
...
+* If an admin added redirect URIs that reached the maximum character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
+
+* When an admin viewed a user's profile, an error sometimes occurred. (OKTA-558220)
+
 ### Weekly release 2022.12.1
 
 | Change | Expected in Preview Orgs |
 |--------------------------------------------------------------------------|--------------------------|
-| [Bugs fixed in 2022.12.1](#bugs-fixed-in-2022-12-1)                      |December 14, 2022            |
+| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                      |December 21, 2022            |
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:27]:
```suggestion
| [Bugs fixed in 2022.12.1](#bugs-fixed-in-2022-12-1)                      |December 14, 2022            |
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md (hunk 3 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:21]: Confirmed with Susan that this is correct.

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 7]
```diff
@@ -6,11 +6,25 @@ title: Okta Identity Engine API Products release notes 2022
...
+* If an admin added redirect URIs and reached the character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
+
+* When an admin viewed a user's profile, an error sometimes occurred. (OKTA-558220)
+
 ### Weekly release 2022.12.1
 
 | Change | Expected in Preview Orgs |
 |--------------------------------------------------------------------------|--------------------------|
-| [Bugs fixed in 2022.12.1](#bugs-fixed-in-2022-12-1)                      |December 14, 2022            |
+| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                      |December 21, 2022            |
```
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md]: error. fixed.

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 8]
```diff
@@ -4,11 +4,23 @@ title: Okta API Products release notes 2022
...
+
 ### Weekly release 2022.12.1
 
 | Change | Expected in Preview Orgs |
 |--------------------------------------------------------------------------|--------------------------|
-| [Bug fixed in 2022.12.1](#bug-fixed-in-2022-12-1)                         |December 14, 2022            |
+| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                         |December 21, 2022            |
```
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022/index.md]: error. fixed.

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 9]
```diff
@@ -4,11 +4,23 @@ title: Okta API Products release notes 2022
...
+* PAttempts to save sign-in page edits sometimes failed when using the full-feature code editor. (OKTA-551632)
+
+* If an admin added redirect URIs that reached the maximum character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
+
 ### Weekly release 2022.12.1
 
 | Change | Expected in Preview Orgs |
 |--------------------------------------------------------------------------|--------------------------|
-| [Bug fixed in 2022.12.1](#bug-fixed-in-2022-12-1)                         |December 14, 2022            |
+| [Bugs fixed in 2022.12.1](#bugs-fixed-in-2022-12-1)                         |December 21, 2022            |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:23]:
```suggestion
| [Bug fixed in 2022.12.1](#bug-fixed-in-2022-12-1)                         |December 14, 2022            |
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 10]
```diff
@@ -4,11 +4,23 @@ title: Okta API Products release notes 2022
...
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                         |December 21, 2022            |
+
+#### Bugs fixed in 2022.12.2
+
+* PAttempts to save sign-in page edits sometimes failed when using the full-feature code editor. (OKTA-551632)
+
+* If an admin added redirect URIs that reached the maximum character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:17]:
```suggestion
* If an admin added a redirect URI that reached the character limit, then they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
```
Minor update approved by @miaopan-okta .

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 11]
```diff
@@ -6,6 +6,20 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                      |December 21, 2022            |
+
+#### Bugs fixed in 2022.12.2
+
+* Attempts to save sign-in page edits sometimes failed when using the full-feature code editor. (OKTA-551632)
+
+* If an admin added redirect URIs that reached the maximum character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:19]:
```suggestion
* If an admin added a redirect URI that reached the character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
```

**Review by vanngo-okta — COMMENTED** [review]
some updates

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 12]
```diff
@@ -4,6 +4,18 @@ title: Okta API Products release notes 2022
...
 
+### Weekly release 2022.12.2
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                         |December 21, 2022            |
+
+#### Bugs fixed in 2022.12.2
+
+* PAttempts to save sign-in page edits sometimes failed when using the full-feature code editor. (OKTA-551632)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:15]:
```suggestion
* Attempts to save sign-in page edits sometimes failed when using the full-feature code editor. (OKTA-551632)
```

**Review by vanngo-okta — APPROVED** [review]
LGTM! 🚀

**Pull request merged**